Dockerfile

```diff
@@ -1,44 +1,69 @@
 # Sets up ai-arena client
+ARG PYTHON_VERSION=3.9
+ARG SC2_VERSION=4.10
+ARG VERSION_NUMBER=1.0.0
+ARG USE_SQUASHED
 
-FROM aiarena/sc2-linux-base:latest
+FROM aiarena/sc2-linux-base:py_$PYTHON_VERSION-sc2_$SC2_VERSION-v$VERSION_NUMBER$USE_SQUASHED
 MAINTAINER AI Arena <staff@aiarena.net>
 
-# Create a symlink for the maps directory
-# Remove the Maps that come with the SC2 client
-RUN ln -s /root/StarCraftII/Maps /root/StarCraftII/maps && rm -Rf /root/StarCraftII/maps/*
+# Debugging purposes
+RUN echo $PYTHON_VERSION
+RUN echo $SC2_VERSION
+RUN echo $VERSION_NUMBER
+RUN echo $USE_SQUASHED
 
-# Calls for a random number to break the cahing of the git clone
-# (https://stackoverflow.com/questions/35134713/disable-cache-for-specific-run-commands/58801213#58801213)
-ADD "https://www.random.org/cgi-bin/randbyte?nbytes=10&format=h" skipcache
+WORKDIR /root/
+
+# Prevent caching unless client master branch changed
```

Comment:
Is this a typo?
Should it say "Prevent caching **when** client master branch changed"?

If so, very useful - been wanting a way to do this.

Reply:
Yes, this is a typo